pages/linux/dnf-automatic.md

```diff
@@ -0,0 +1,16 @@
+# dnf-automatic
+
+> A tool for automating the checking, downloading, and/or applying package updates via dnf.
+> More information: <https://dnf.readthedocs.io/en/latest/automatic.html>.
+
+- Override config file and only notify of available updates:
+
+`sudo systemctl enable --now dnf-automatic-notifyonly.timer`
+
+- Run with default config file:
+
+`sudo systemctl enable --now dnf-automatic.timer`
+
+- Use dnf-automatic in a custom systemd timer:
+
+`/usr/bin/dnf-automatic {{/path/to/automatic.conf}} --timer`
```

Comment:
In this example, the command should be in the `$PATH`, so the full path is not required here:
```suggestion
`dnf-automatic {{path/to/automatic.conf}} --timer`
```